acrobat/scripts/scripts.js

```diff
@@ -338,6 +338,10 @@ if (IMS_GUEST) {
   };
 }
 
+// Load IMS
+const { default: dcLoadIMS } = await import('./dcIMS.js');
```

Comment:
also FYI this import adds delay to the startup, so we generally try to inline code here or make the inlining happen in the edgeworker as needed